src/main/resources/db/migration/V1_2020_001.sql

```diff
@@ -0,0 +1,81 @@
+CREATE TABLE trustgroup (
```

Comment:
Incorrect migration file and naming. I guess the migrations are not running at all.

Reply:
Add `spring.jpa.hibernate.ddl-auto=validate` to your properties